src/oc/change/resources/seen.clj

```diff
@@ -24,21 +24,30 @@
     container-id :- lib-schema/UniqueID
     item-id :- lib-schema/UniqueID
     seen-at :- lib-schema/ISO8601]
-  (far/put-item c/dynamodb-opts table-name {
-      :user_id user-id
-      :container_item_id (str container-id "-" item-id)
-      :container_id container-id
-      :item-id item-id
-      :user-id user-id
-      :seen_at seen-at
-      :ttl (coerce/to-long (time/plus (time/now) (time/days c/seen-ttl)))})
+  (let [;; If ttl value is set from env var it's a string, if it's default is an int
+        fixed-seen-ttl (if (string? c/seen-ttl)
```

Comment:
Couldn't you move this block of code to the utils namespace?  Since it is repeated https://github.com/open-company/open-company-change/pull/11/files#diff-12699910059652a231ec6d1184b1590fR30